.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "iwf-idl"]
 	path = iwf-idl
-	url = git@github.com:cadence-oss/iwf-idl.git
+	url = git@github.com:indeedeng/iwf-idl.git
```

Comment:
updated this because it's blocking updating idl

Reply:
I think you also need to commit the commitId of the module

Reply:
there is update of iwf-idl submodule change in this PR, but I'm not sure why it's not showing the original commit id and the new commit id, instead it's showing the changes in the file. And if you click into the change, both commit are in the same repo

is it because submodule url is changed?